### ISSUE TITLE
HTML generated code is wrongly formatted

### ISSUE THREAD

**Issue by SteelPh0enix** [goessner/mdmath#58]
### Summary

When i generate HTML code of my LaTeX+MD document, sqrt symbol in fraction is moved (misaligned)

### Expected behavior

This should look like that (screenshot from VS Code internal preview)
![gog galaxy notifications renderer_2018-09-08_15-04-11](https://user-images.githubusercontent.com/10062791/45254437-78dd0c00-b378-11e8-9343-5d906174b1a1.png)

### Actual behavior

But when i generate HTML with Ctrl+K . command, paste it into HTML file and open in Chrome, it looks like that
![chrome_2018-09-08_15-01-50](https://user-images.githubusercontent.com/10062791/45254430-4af7c780-b378-11e8-9508-62598003296b.png)

### Steps to reproduce

1. Put code from example below into markdown file:
2. Generate HTML code with Markdown+Math
3. Put code into HTML file and open it. You should see this: 
![image](https://user-images.githubusercontent.com/10062791/45254458-ceb1b400-b378-11e8-8e69-b9310fe0df5c.png)

### Code example

```md
# Test

$$ f'(1) = \lim_{x \to 1} \frac{f(x) - f(1)}{x - 1} = \lim_{x \to 1} \frac{\sqrt{x^2 + 3} - \sqrt{1^2 + 3}}{x - 1} = \lim_{x \to 1} \frac{\sqrt{x^2 + 3} - 2}{x - 1} = \lim_{x \to 1} \left( \frac{\sqrt{x^2 + 3} - 2}{x - 1} \cdot \frac{\sqrt{x^2 + 3} + 2}{\sqrt{x^2 + 3} + 2} \right) = \lim_{x \to 1} \frac{x^2 + 3 - 4}{(x - 1)\sqrt{x^2 + 3} + 2} = $$
```

### Related links

* [Markdown+Math](https://github.com/goessner/mdmath)

### Environment

```text
     Operating system: Windows 10
       VSCode version: 1.27.1
Markdown+Math version: 2.3.3
```

**Comment by goessner** [comment]
Hmm .. do you know, if it looked correct with version version 2.3.2 and below .. ?

**Comment by goessner** [comment]
Exported html had outdated link to `katex.min.css` ... resolved with `mdmath 2.3.4`.

Thanks
